### ISSUE TITLE
Regression error using graphql v14

### ISSUE THREAD

**Issue by emmanuelbuah** [dotansimha/graphql-code-generator#556]
There's a regression error that surfaces using graphql v14

```
TypeError: Cannot read property 'slice' of undefined
    at buildDirective (../node_modules/graphql/utilities/buildClientSchema.js:286:51)
    at Array.map (<anonymous>)
    at Object.buildClientSchema (../node_modules/graphql/utilities/buildClientSchema.js:302:84)
    at Object.introspectionToGraphQLSchema (../node_modules/graphql-codegen-core/dist/utils/introspection-to-schema.js:11:22)
    at Request._callback (../node_modules/graphql-code-generator/dist/loaders/schema/introspection-from-url.js:59:48)
    at Request.self.callback (../node_modules/graphql-code-generator/node_modules/request/request.js:185:22)
    at Request.emit (events.js:182:13)
    at Request.<anonymous> (../node_modules/graphql-code-generator/node_modules/request/request.js:1161:10)
    at Request.emit (events.js:182:13)
    at IncomingMessage.<anonymous> (../node_modules/graphql-code-generator/node_modules/request/request.js:1083:12)
```

This looks like its due to the breaking change by https://github.com/graphql/graphql-js/pull/1429/. More on changes in graphql v14 at https://github.com/graphql/graphql-js/issues/1383

**Comment by IvanGoncharov** [comment]
@emmanuelbuah It's because of this change https://github.com/graphql/graphql-js/pull/1429
This introspection format was deprecated for last few years so we removed it in `14.0.0` release.
What library do you use to implement GraphQL on your backend?

**Comment by emmanuelbuah** [comment]
@IvanGoncharov  I'm using apollo express server  v1.4

**Comment by IvanGoncharov** [comment]
@emmanuelbuah Looks like everything good with your server and the bug happening because of this line:
https://github.com/dotansimha/graphql-code-generator/blob/8c071a8076a0ce5f58ae14347ae1512eab02deb6/packages/graphql-codegen-cli/src/loaders/schema/introspection-from-url.ts#L52
I'm not sure why this library needs to remove directive locations.

It can't be fixed in 'graphql-js' but I will add explicit check and better error message on the next release.

**Comment by dotansimha** [comment]
Fixed in 0.12.0
